api/v1alpha1/releaseplan_types.go

```diff
@@ -47,7 +47,7 @@ type ReleasePlanSpec struct {
 
 	// Target references where to send the release requests
 	// +kubebuilder:validation:Pattern=^[a-z0-9]([-a-z0-9]*[a-z0-9])?$
-	// +required
+	// +optional
```

Comment:
does this require any special handling in the controller since it might expect this is required?

Reply:
hmm probably? but I'm not sure we need to handle it yet; it can likely be handled in the PR that adds support for the dev pipelines. Maybe this is better off as part of that PR instead. @davidmogar thoughts?

Reply:
Yeah, probably better to add it there.